language/move-lang/tests/functional/translated_ir_tests/transaction_fee_distribution/test_txn_fee_collection.move

```diff
@@ -0,0 +1,40 @@
+//! account: alice, 90000
+
+//! sender: alice
+//! gas-price: 1
+
+use 0x0::LibraAccount;
+use 0x0::LBR;
+use 0x0::TransactionFeeAccounts;
+use 0x0::Transaction;
+
+fun main() {
+    let x = 10;
+    // Make sure that the pot is empty to start with
+    let fee_addr = TransactionFeeAccounts::transaction_fee_address<LBR::T>();
+    let transaction_fee_pot_amount = LibraAccount::balance<LBR::T>(fee_addr);
+    Transaction::assert(transaction_fee_pot_amount == 0, 0);
+    // Do some work
+    while (x > 0) {
+        x = x - 1;
+    }
+}
+
```

Comment:
Nit: `// check: EXECUTED` (here and elsewhere)? I'm so paranoid about this after being bitten by weird matcher behavior...